scripts/cicd/custom-nodes-summary.py

```
@@ -0,0 +1,115 @@
+
+data = json.load(open('custom-nodes-results.json'))
+stats = data.get('stats', {})
+
+TIERS = ['startup/load', 'all nodes', 'curated run', 'dynamic inputs', 'interaction']
+packs, wide = {}, {}
+
+def bucket(file, path):
+    title = ' > '.join(path)
+    pack = None
+    m = re.search(r'(?:custom node|all nodes|dynamic inputs|interaction profiles): ([^@>]+?)(?:\s*@|\s*>|$)', title)
+    if m: pack = m.group(1).strip()
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)